maven-plugin/plugin-core/src/main/java/org/jvnet/jaxb/maven/AbstractXJCMojo.java

```diff
@@ -41,6 +41,14 @@
 public abstract class AbstractXJCMojo<O> extends AbstractMojo implements
 		DependencyResourceResolver {
 
+    /**
+     * This method should be overriden in extending classes to get real value
+     * @return currently running XJC version
+     */
+    public XJCVersion getVersion() {
+        return XJCVersion.empty();
```

Comment:
How about we make this a XJCVersion.NOT_DETECTED; <-- static string?

Reply:
Will define static instance UNDEFINED